### PR TITLE
Mogo

### PR DESCRIPTION
**Pull request by KamilWasilonek**
https://kamilwasilonek.github.io/ui-internship-lwo2/html/Mogo/

### CODE REVIEW

File: html/Mogo/index.html [hunk 1]
```diff
@@ -0,0 +1,1084 @@
...
+  <section class="banner">
+    <!-- Menu -->
+    <header>
+      <nav class="nav banner__nav">
+        <!-- Logo -->
+        <a href="#" class="nav__logo">MoGo</a>
+        
+        <!-- Menu list -->
+        <ul class="nav__menu">
+          <li class="nav__menu-item">
```
Comment (tkachukivan) [inline html/Mogo/index.html]: not sure about this one, but with naming like this menu item is a part of nav but should be part of nav-menu

File: html/Mogo/index.html [hunk 2]
```diff
@@ -0,0 +1,1084 @@
...
+          <div class="moving-image__overlay-content">
+            <img src="./assests/images/about/users.svg" alt="">
+            <p class="small-title">Super team</p>
+          </div>
+        </div>
+
+      </div>
+
+
+
```
Comment (tkachukivan) [inline html/Mogo/index.html]: too many spaces :D

File: html/Mogo/index.html [hunk 3]
```diff
@@ -0,0 +1,1084 @@
...
+     
+
+      
+    </div>
+
+  </section>
+
+  <!-- Counters -->
+  <section class="counters">
+    <div class="counters__container">
```
Comment (tkachukivan) [inline html/Mogo/index.html]: I would say that you have counter here and after counter__number/counter__details

File: html/Mogo/index.html [hunk 4]
```diff
@@ -0,0 +1,1084 @@
...
+    
+    <!-- Title -->
+    <h2 class="section-title section-title--black unique-design__title">
+      Unique design
+    </h2>
+
+    <!-- Images -->
+    <div class="unique-design__container">
+      <!-- Tablet -->
+      <img class="unique-design__tablet" src="./assests/images/unique-design/display-bg-768-x-1024@2x.jpg" alt="Tablet view">
```
Comment (tkachukivan) [inline html/Mogo/index.html]: alt not describe that you have in image and I'm just interested if there is other way to do it without two img tags?

File: html/Mogo/index.html [hunk 5]
```diff
@@ -0,0 +1,1084 @@
...
+      <img class="unique-design__mobile" src="./assests/images/unique-design/white@2x.png" alt="Mobile view">
+    </div>
+   
+  </section>
+
+  <!-- Services -->
+  <section class="section services">
+
+    <!-- Type -->
+    <p class="section-type section-type--black services__type">
```
Comment (tkachukivan) [inline html/Mogo/index.html]: I would say that it is h* more than p

File: html/Mogo/index.html [hunk 6]
```diff
@@ -0,0 +1,1084 @@
...
+        <!-- Article 1 -->
+        <article class="article latest-posts__article">
+          <!-- Image -->
+          <div class="article__image">
+            <img class="article__img" src="./assests/images/latest-posts/layer-54@3x.jpg" alt="Creativity">
+            <div class="article__date"> <span>15</span>Jan</div>
+          </div>
+
+          <!-- Title -->
+          <h4 class="small-title article__title">
```
Comment (tkachukivan) [inline html/Mogo/index.html:970]: usually article heading also a link by which you can go and read full article

File: html/Mogo/styles/components.scss [hunk 7]
```diff
@@ -0,0 +1,606 @@
...
+  &::after {
+    content: "";
+    display: block;
+    width: 60px;
+    height: 2px;
+    // margin: 20px auto;
+    background: $separator-color;
+
+    @include respond(lg) {
+      // margin: 40px auto;
```
Comment (tkachukivan) [inline html/Mogo/styles/components.scss]: so maybe possible to remove? :)

File: html/Mogo/styles/components.scss [hunk 8]
```diff
@@ -0,0 +1,606 @@
...
+      color: $white-color;
+      font-size: 0.8rem;
+      text-decoration: none;
+      text-transform: uppercase;
+
+      &:hover {
+        color: $menu-on-hover-color;
+      }
+
+      &:hover::after {
```
Comment (tkachukivan) [inline html/Mogo/styles/components.scss]: why not just add border? :)
sam applies below

File: html/Mogo/styles/components.scss [hunk 9]
```diff
@@ -0,0 +1,606 @@
+@import "./variables.scss";
```
Comment (tkachukivan) [inline html/Mogo/styles/components.scss:1]: probably it is possible to split this file to smaller files

File: html/Mogo/styles/sections/counters.scss [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+  @include respond(lg) {
+    flex-wrap: nowrap;
+    padding: 0 10%;
+
+    div,
+    div:nth-child(even) {
+      border: none;
+    }
+
+    div:nth-child(odd) {
```
Comment (tkachukivan) [inline html/Mogo/styles/sections/counters.scss:37]: sometimes is better to use classes instead of nth-child and so on

**Review by tkachukivan — APPROVED** [review]

**Review by tsvetkovv — APPROVED** [review]
good job

**Pull request merged**